### DISCUSSION TITLE
How is shutter speed calculated with EXPT in timelapse video?

### DISCUSSION THREAD

**Discussion by kouyoumin** [gopro/labs#487] [Q&A]
A new feature in the 2.12.70 release note says:
`- Enhanced EXTQ,EXTN,EXTX and EXPT to work on timelapse video and photo`
(I believe EXT~ should be EXP~)

As I know, `exposure_time = (shutter_angle / 360) / fps`
I wonder how to determine fps in timelapse video.
Is it based on capture speed (say, 1 fps) or playback speed (30 fps)?
Will speed ramp change exposure time?

**Comment by dnewman-gpsw** [comment]
Thanks, I fixed the typos.

I'm not sure if EXPT will work for timelapse.  Try it.  Let me know.  There are too many combinates to test even 10% of what Labs can do.

EXPQ/EXPN/EXPX should all work, as that are simple shutter speeds in form 1/x.

So if you shoot indoor under 50Hz lighting, and want a timelapse without any flicker, you can set oMEXPQ=100. The min and max shutter range are both 1/100, so the ISO is the only way the camera can set exposure.

**Reply by qlyoung** [reply]
The website is still showing the typos, FYI.

I'm not sure if you heard from support or this was just a coincidence, but I had actually contacted GoPro support and requested this feature (EXPN/EXPX working in photo mode) [on August 4th](https://www.reddit.com/r/gopro/comments/15hcvfh/comment/juu7xl5/?utm_source=reddit&utm_medium=web2x&context=3). I saw the new firmware today and just wanted to drop a line and say thank you so much for turning it around in 6 days, that is just ridiculously awesome. Labs rocks and you folks are doing a killer job adding so much value to the cameras.

**Reply by dnewman-gpsw** [reply]
Thanks, I just pushed the changes to finally fix those typos. 

Labs requests are always best posted here, more direct than going through support.
